tests/financial/test_india_stock_exchange.py

```diff
@@ -0,0 +1,299 @@
+#  holidays
```

Comment:
Check out [`test_anguilla.py`](https://github.com/vacanza/holidays/blob/c4b527c344adf15a9b046e6361acee2ab9570ea5/tests/countries/test_anguilla.py) for examples